mail-client/alot/alot-0.9.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+SRC_URI="https://github.com/pazz/alot/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="
+	app-crypt/gpgme[python,${PYTHON_USEDEP}]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/python-magic[${PYTHON_USEDEP}]
+	dev-python/urwid[${PYTHON_USEDEP}]
+	dev-python/urwidtrees[${PYTHON_USEDEP}]
+	>=dev-python/twisted-18.4[${PYTHON_USEDEP}]
+	net-mail/mailbase
+	net-mail/notmuch[crypt,python]
```

Comment:
I still think you'll need to add `PYTHON_USEDEP` here, the test phase clearly implies that. However with notmuch-30 I'm not sure how it affects the runtime, you probably have a better idea? How is notmuch used by alot? It should be broken with 30 currently. Maybe the version can be restricted for now, or we wait until notmuch-30 has been fixed?

Reply:
> I still think you'll need to add PYTHON_USEDEP here, the test phase clearly implies that. However with notmuch-30 I'm not sure how it affects the runtime, you probably have a better idea?

I am not really sure, what I can say alot use the python binding provided by notmuch,
I used to think that when notmuch is installed it is build with all python version, maybe I am wrong.

> Maybe the version can be restricted for now, or we wait until notmuch-30 has been fixed?
Yes I contacted upstream (https://github.com/pazz/alot/issues/1539) it seems,
a PR is already going on to port to 'notmuch2' module (https://github.com/pazz/alot/pull/1511).

So with all that, I think I probably fix to '<=net-mail/notmuch-0.30[crypt,python,${PYTHON_USEDEP}]',
until everything is fixed.